lib/Runtime/Library/JavascriptBigInt.h

```diff
@@ -0,0 +1,71 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+#pragma once
+
+namespace Js
+{
+    class JavascriptBigInt sealed : public RecyclableObject
+    {
+
+    private:
+        Field(uint32*) m_digits;         // digits
+        Field(uint32) m_length;          // length
```

Comment:
Is the specced max length UINT_MAX? What do other engines do here?

Reply:
It is only restricted by memory space. Currently we allow string length to INT_MAX-1 so we can do the same here.